package.json

```diff
@@ -33,8 +34,11 @@
     "browserify": "^13.3.0",
     "eslint": "^3.14.0",
     "eslint-config-homeoffice": "^2.1.0",
+    "funkie": "0.0.5",
+    "funkie-phantom": "0.0.1",
     "nodemon": "^1.11.0",
-    "npm-run-all": "^4.0.1"
+    "npm-run-all": "^4.0.1",
```

Comment:
I know it isn't *actually* part of this diff, but this doesn't look to be being used, so can (and should) be removed.